### ISSUE TITLE
monoOne text cut off in awful widgets

### ISSUE THREAD

**Issue by Ginto8** [awesomeWM/awesome#395]
I'm using [monoOne](/madmalik/monoOne) in Awesome, and all the text gets cut off on the left, like in [this picture of my taglist](https://cloud.githubusercontent.com/assets/1918771/9128268/90cea5da-3c93-11e5-99da-739907f7d933.png). This happens in all of my widgets, and does not occur with the default font. This was a recent change, but I don't know what exact version I was using before. I'm running Arch, and my awesome version is:
```
awesome v3.5.6 (For Those About To Rock)
 • Build: May  5 2015 05:45:29 for x86_64 by gcc version 5.1.0 (builduser@)
 • Compiled against Lua 5.3.0 (running with Lua 5.3)
 • D-Bus support: ✔
```

**Comment by actionless** [comment]
```
xrdb -query  | grep dpi
```

i guess what because of recent change related to reading dpi from xresources you font became bigger and not suites panel size

**Comment by blueyed** [comment]
@actionless 
> recent change related to reading dpi from xresources

That has not been released yet (as in 3.5.6).

@Ginto8 
I assume it does not happen with another font?

Does it happen with a normal textbox, too (if you create a wibox with just a textbox), or is it specific to the taglist?
Are other widgets like the tasklist affected?

**Comment by Ginto8** [comment]
@blueyed It works fine with [courier](https://cloud.githubusercontent.com/assets/1918771/9135607/5e393834-3cdd-11e5-8541-38ff5b173642.png), and yes, it affects all text boxes. My tasklist is unaffected though -- maybe it only cuts off text at the far left boundary of a box? Those pipe separators are just text boxes containing "&lt;b&gt;|&lt;/b&gt;", and if I switch back to monoOne, I get
[this](https://cloud.githubusercontent.com/assets/1918771/9135652/a59649ec-3cdd-11e5-8001-bbe74c58be3d.png). If I add a space before the pipe in the separators (ie. "&lt;b&gt; |&lt;/b&gt;"), the pipes [reappear](https://cloud.githubusercontent.com/assets/1918771/9135684/0c625634-3cde-11e5-8cb8-dbc7f586ec49.png).

**Comment by psychon** [comment]
Cannot reproduce with v3.5.2-622-gbaeb8fd-dirty (basically "git master"), the default config and the sky theme:
![screenshot](https://cloud.githubusercontent.com/assets/89482/9136277/9c85c188-3d15-11e5-9f86-4ebf870030b3.png)

Cannot reproduce with v3.5.6 either (default config, default theme, as packaged by debian testing, only changed theme.font to "monoOne 8"):
![screenshot2](https://cloud.githubusercontent.com/assets/89482/9136314/f9af8164-3d15-11e5-8609-49ff4947b359.png)

**Comment by Ginto8** [comment]
To rule out my theme and config, I reverted to default config and default theme, and reinstalled monoOne with the latest version (I noticed I had an old version installed). [Same issue](https://cloud.githubusercontent.com/assets/1918771/9145674/7da19a26-3d24-11e5-8333-40c8d8bfeee0.png). Perhaps it's not awesome itself, but a library it depends on? If someone else with Arch could attempt to reproduce, we could narrow it down a bit.
